packages/modules/modules.js

```diff
@@ -0,0 +1,27 @@
+meteorInstall = makeInstaller();
+
+meteorInstall({
+  node_modules: {
+    modules: {
+      "css.js": function (r, exports) {
```

Comment:
What is this? This lets users call `require("css.js").addStyles("...css text...")`? I couldn't find it used anywhere else in the code.